tests/main/xdg-runtime-dir/task.yaml

```diff
@@ -0,0 +1,16 @@
+summary: Ensure that XDG_RUNTIME_DIR is created
+
+details: |
+    Ensure that the XDG_RUNTIME_DIR is created when a snap is launched.
+
+
+execute: |
+    echo Testing XDG_RUNTIME_DIR
+    snap install test-snapd-xdg-dir
+    ls /run/user/
```

Comment:
Why the `ls /run/user`? Is this to verify the path exists? If that is, then please use a more explicit check where the intent is more clear, like `test -e /run/user`.

Reply:
Ops... No, that are some remains from an old test. Removing.